docs/resources/telemetry_provider.md

```diff
@@ -0,0 +1,98 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "ybm_telemetry_provider Resource - ybm"
+subcategory: ""
+description: |-
+  The resource to create a telemetry provider config in YugabyteDB Managed.
+---
+
+# ybm_telemetry_provider (Resource)
+
+The resource to create a telemetry provider config in YugabyteDB Managed.
+
+## Example Usage
+
+```terraform
```

Comment:
should be injected, are your sure you edit a template?

Reply:
These changes are auto generated upon doing `make doc`. Any updates I need to do here?

Reply:
Just use set a template, as other here. Will bee easier to customize when needed. Pus how do wee migrate frorm metrics exporter ?
https://github.com/yugabyte/terraform-provider-ybm/tree/CLOUDGA-19630/templates

Reply:
1. Updated the template
2. Telemetry provider is just a name change. All the fields under the telemetry provider are exactly the same to ensure hassle free migration